build.gradle.kts

```diff
@@ -2,17 +2,19 @@ import com.github.jk1.license.filter.LicenseBundleNormalizer
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.springframework.boot") version "3.0.6"
-    id("io.spring.dependency-management") version "1.1.0"
-    kotlin("jvm") version "1.8.0"
-    kotlin("plugin.spring") version "1.8.0"
-    id("com.diffplug.spotless") version "6.18.0"
+    alias(libs.plugins.org.springframework.boot)
+    alias(libs.plugins.io.spring.dependency.management)
+    alias(libs.plugins.org.jetbrains.kotlin.jvm)
+    alias(libs.plugins.org.jetbrains.kotlin.plugin.spring)
+    alias(libs.plugins.com.diffplug.spotless)
     id("jacoco")
     id("jacoco-report-aggregation")
-    id("org.sonarqube") version "4.0.0.2929"
-    id("com.github.jk1.dependency-license-report") version "2.1"
-    id("com.adarshr.test-logger") version "3.2.0"
-    id("com.gorylenko.gradle-git-properties") version "2.4.0"
+    alias(libs.plugins.org.sonarqube)
+    alias(libs.plugins.com.github.jk1.dependency.license.report)
+    alias(libs.plugins.com.adarshr.test.logger)
+    alias(libs.plugins.com.gorylenko.gradle.git.properties)
+    alias(libs.plugins.com.github.ben.manes.versions)
+    alias(libs.plugins.nl.littlerobots.version.catalog.update)
```

Comment:
Do we need both of those plugins for the new update task?

Reply:
The first plugin provides the `dependencyUpdates` task - unrelated to version catalogs, checks all dependencies for newer versions in the configured maven repositories, and creates a report.
The second plugin provides the `versionCatalogUpdate` task, which uses the data about newer versions from the first plugin, and updates the version catalog accordingly. It also provides a task to auto-format and sort the version catalog (`versionCatalogFormat`).

Reply:
Thank you 🙌